public/js/post/post.js

```diff
@@ -4,11 +4,28 @@ var CourseCode = React.createClass({
     },
 
     componentWillMount: function() {
-        this.setState({selected: getCookie(this.props.courseID) === 'active'});
+        this.setState({selected: getCookie(this.props.courseIDs[0]) === 'active'});
     },
 
     toggleFullInfo: function() {
-        $('#' + this.props.courseID + '_info').toggle();
+        $('#' + this.props.courseIDs[0] + '_info').toggle();
```

Comment:
You should just use the component's state to handle the visibility of the element, instead of relying on DOM manipulation.